app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -62,7 +62,7 @@ $font-size-base: 13px;
 // $font-size-h6:            ceil(($font-size-base * 0.85)) // ~12px
 
 //** Unit-less `line-height` for use in components like buttons.
-// $line-height-base:        1.428571429 // 20/14
+$line-height-base: 1.538461; // 20/13
```

Comment:
http://stackoverflow.com/a/20123813

Bootstrap is aiming at a 20px line height for a 14px font-size. I am not sure if we would also like to aim at a 20px line-height or keep the current value.

It would be great if you could remove this change from the PR and move it to a separate one. All other changes in this PR are needed to merge the BS3 branch into develop and I'd like to get a few opinions from different devs about the line-height change since the line-height is used to compute all kinds of stuff for BS3.